src/Services/EmailService.php

```diff
@@ -90,13 +91,27 @@ public function createStalledJobReport(string $subject, string $message, int $jo
      */
     public function createReport(string $subject): ?Email
     {
-        $from = Config::inst()->get(Email::class, 'admin_email');
-        $to = Config::inst()->get(Email::class, 'queued_job_admin_email');
+        $from = $this->extractEmailAddress(Config::inst()->get(Email::class, 'admin_email'));
+        $to = $this->extractEmailAddress(Config::inst()->get(Email::class, 'queued_job_admin_email'));
 
         if (!$to) {
             return null;
         }
 
         return Email::create($from, $to, $subject);
     }
+
+    /**
+     * Handle ['admin@mysite' => 'My admin name'] style email configuration
+     */
+    private function extractEmailAddress(string|array $email): string
+    {
+        if (is_array($email)) {
+            if (count($email) === 0) {
+                throw new InvalidArgumentException('Empty array passed for $email');
+            }
+            return array_keys($email)[0];
```

Comment:
This will essentially just throw away the "name" part of the email address, which is not the expected behaviour.

Reply:
Having a look through https://github.com/silverstripe/silverstripe-framework/blob/5/src/Control/Email/Email.php - looks like we may have accidentally lost support for the "Name" portion of array config during the upgrade to symfony/mailer e.g. https://github.com/silverstripe/silverstripe-framework/blob/5/src/Control/Email/Email.php#L206 will also remove the "Name" portion

Given that we should probably add support back in to Email rather than fixing this here